hw/top_earlgrey/doc/design/index.md

```diff
@@ -177,12 +177,32 @@ See the [pinmux specification]({{< relref "hw/ip/pinmux/doc" >}}) for how to con
 
 ##### SPI device
 
-**TODO: this section has pending updates**
 
-The SPI device implements Firmware Mode, a feature that provides the ability for external drivers to send firmware upgrade code into a bank of embedded flash memory for in-field firmware updates.
-Firmware mode has no addressing, and at the moment no other addressing modes are provided.
+The SPI device implements multiple modes:
+- Firmware mode
+- TPM mode
+- Flash mode
+- Passthrough mode
 
-See the [SPI device specification]({{< relref "hw/ip/spi_device/doc" >}}) for more details on the Firmware Mode implementation.
+Firmware Mode is a generic data transfer mode that.
```

Comment:
it seems that `that` is not needed at the end of this sentence...

Reply:
i think i forgot to remove this line...

Reply:
o wait i see what you're saying.